Frameworks/CoreText/CTLine.mm

```diff
@@ -248,12 +242,8 @@ void CTLineDraw(CTLineRef lineRef, CGContextRef ctx) {
  @Status Interoperable
 */
 CFIndex CTLineGetGlyphCount(CTLineRef lineRef) {
-    if (lineRef == nil) {
-        return 0;
-    }
-
     _CTLine* line = (_CTLine*)lineRef;
-    return (line->_strRange).length;
+    return line ? (line->_strRange).length : 0;
```

Comment:
>(line->_strRange).length [](start = 18, length = 24)

I just learnt that character count != glyph count for all languages. We should get the total glyph count from the glyph runs that actually make the line. May be it is worth caching this as a new member in _CTLine